libs/design/src/core/colorable/colorable.spec.ts

```diff
@@ -55,6 +55,29 @@ describe('daffColorMixin', () => {
         }).toThrow(new TypeError("SOMEFAKEPALETTE is not a valid color for the DaffPalette"));
 
     });
+
```

Comment:
I don't think we should use the word "if" or "when" in an `it`. We should use `describe` to handle that. By doing this, the code becomes more test driven, and I think it becomes a whole lot easier for a new person to come in and digest both the tests and the code it is testing, because it's easier to read the spec file as a whole rather than test by test. Also, if we add more tests later that share an "if" or "when" with an existing test, we won't need to refactor it into a describe. But mostly because of the TDD reason.